src/test/regress/expected/failure_connection_establishment.out

```diff
@@ -325,12 +325,26 @@ SELECT citus.mitmproxy('conn.onQuery(query="^SELECT citus_check_connection_to_no
 SELECT * FROM citus_check_cluster_node_health();
  from_nodename | from_nodeport | to_nodename | to_nodeport | success
 ---------------------------------------------------------------------
- localhost     |          9060 | localhost   |        9060 | f
- localhost     |          9060 | localhost   |       57637 | f
+ localhost     |          9060 | localhost   |        9060 |
+ localhost     |          9060 | localhost   |       57637 |
  localhost     |         57637 | localhost   |        9060 | t
  localhost     |         57637 | localhost   |       57637 | t
 (4 rows)
 
+-- note that it is not safe to use bool_and aggregate when you have null values because bool_and(NULL, true) evaluates to true.
+SELECT bool_and(success) FROM citus_check_cluster_node_health();
+ bool_and
+---------------------------------------------------------------------
+ t
+(1 row)
+
+SELECT success, count(*) FROM citus_check_cluster_node_health() GROUP BY success ORDER BY 1;
+ success | count
+---------------------------------------------------------------------
+ t       |     2
+         |     2
+(2 rows)
+
```

Comment:
@onderkalaci it is no longer feasible to use `bool_and()` to create a summary of the connectivity check. Here is an example, why that is necessary.

Please let me know if you think we should not have these lines in failure tests.